mobile/wp8/wp8-emulator-services.ts

```diff
@@ -20,6 +20,11 @@ class Wp8EmulatorServices implements Mobile.IEmulatorPlatformServices {
 		return "";
 	}
 
+	public async getRunningEmulatorId(image: string): Promise<string> {
+		//todo: plamen5kov: fix later if necessary
+		return "";
```

Comment:
You should return a Promise here - return Promise.resolve("")

Could you also remove the 'todo' and instead add a comment that it's an empty stub?